internal/gitinterface/keys.go

```diff
@@ -3,147 +3,121 @@
 package gitinterface
 
 import (
-	"errors"
+    "errors",
```

Comment:
There shouldn't be a comma here. Also note that Go plugins should handle imports for you automatically too when you use something from the package in question.